packages/api/src/constants.js

```diff
@@ -32,10 +32,7 @@ switch (CLUSTER_SERVICE) {
 
 export const cluster = {
   apiUrl: clusterUrl,
-  basicAuthToken:
-    typeof CLUSTER_BASIC_AUTH_TOKEN !== 'undefined'
-      ? CLUSTER_BASIC_AUTH_TOKEN
-      : '',
+  basicAuthToken: CLUSTER_BASIC_AUTH_TOKEN,
```

Comment:
Makes this global mandatory...but there is no basic auth by default...

Reply:
there is now with this setup, we already had issues in the past for not testing with auth. I think gozala even added auth to the cluster client tests